_overviews/scaladoc/for-library-authors.md

```diff
@@ -180,7 +180,7 @@ For explicit comment inheritance, use the `@inheritdoc` tag.
 It is still possible to embed HTML tags in Scaladoc (like with Javadoc), but
 not necessary most of the time as markup may be used instead.
 
-Some of the standard markup available:
+Some standard markup available:
```

Comment:
I agree the original is awkward, but the new version is awkward too. How about "Some types of markup available"?

Reply:
Agree